packages/near-fast-auth-signer-e2e-tests/utils/firebase.ts

```diff
@@ -11,16 +11,42 @@ import { serviceAccount } from './serviceAccount';
 
 const FIREBASE_API_KEY_TESTNET = 'AIzaSyDAh6lSSkEbpRekkGYdDM5jazV6IQnIZFU';
 
+export const isServiceAccountAvailable = () => {
+  if (
+    !serviceAccount.private_key_id
+    || !serviceAccount.private_key
+    || !serviceAccount.client_email
+    || !serviceAccount.client_id
+    || !serviceAccount.client_x509_cert_url
+  ) {
+    return false;
+  }
+  return true;
+};
+
 export const initializeAdmin = () => {
-  admin.initializeApp({
-    credential: admin.credential.cert(serviceAccount)
-  });
+  if (isServiceAccountAvailable() && !admin.apps.length) {
+    admin.initializeApp({
+      credential: admin.credential.cert(serviceAccount)
+    });
+  }
 };
 
 export const deleteAccount = async (userUid: string) => {
   await admin.auth().deleteUser(userUid);
 };
 
+export const deleteUserByEmail = async (email: string) => {
```

Comment:
Is it possible to also delete from MPC?

Reply:
sorry I don't quite get what do you mean? this code basically deletes firebase record and that is it.